lib/orbit/evented.js

```diff
@@ -231,8 +231,14 @@ var Evented = {
           if (listeners.length === 0) {
             resolve();
           } else {
-            var listener = listeners.shift(),
-                response = listener[0].apply(listener[1], args);
+            var listener = listeners.shift();
+            var response;
+            try {
+              response = listener[0].apply(listener[1], args);
+            } catch (e) {
+              console.error('Orbit ignored error in event listener', eventNames);
```

Comment:
I kind of want to make an `Orbit.internal.ignoreError(e, 'event listener', eventNames, listener);` for this